components/ws-manager-bridge/src/config.ts

```diff
@@ -38,4 +38,7 @@ export interface Configuration {
 
     // clusterSyncIntervalSeconds configures how often we sync workspace cluster information
     clusterSyncIntervalSeconds: number;
+
+    // maxTimeToRunningPhaseSeconds is the timeout duration
+    maxTimeToRunningPhaseSeconds: number;
```

Comment:
I would love to not expose this as a property, but simply inline a magic number. I don't see why this should be configurable and the config surface is already rather large.

Reply:
Oops I missed to remove the whole part of this config which is not needed anymore 🙏🏽